docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -19,6 +19,10 @@ The main features of the Media Controller API include:
 
   You can [send a request](#managing-requests) from the client to the server to modify various playback attributes. In the server, you can set up a listener to react to the client request and perform the requested task.
 
+- Set server icon URI
+
+  You can [set server icon URI](#set-server-icon) and read it from the client.
```

Comment:
The cross-reference is not working.
Please fix the link after rephrase as:
(#setting-server-icon)

Reply:
done